mmv1/templates/terraform/examples/looker_instance_full.tf.erb

```diff
@@ -0,0 +1,48 @@
+resource "google_looker_instance" "<%= ctx[:primary_resource_id] %>" {
+  name               = "<%= ctx[:vars]["instance_name"] %>"
+  platform_edition   = "LOOKER_CORE_STANDARD"
+  region             = "us-central1"
+  private_ip_enabled = false
```

Comment:
Can we flip public/private here so that we're using values different than the defaults?

Reply:
Done.